x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/overview/overview.tsx

```diff
@@ -291,6 +309,10 @@ export const OverviewPage: React.FC<Props> = memo(
     const requireAgentRootPrivileges = isRootPrivilegesRequired(packageInfo);
     const hideDashboards = config?.hideDashboards;
 
+    const showLogsEssentialsCallout = isEqual(config?.internal?.excludeDataStreamTypes, [
```

Comment:
The description says that it will be needed on serverless, do we want to specify this constraint in the code as well? Not sure if there's a plan to support it on ECH as well.